test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/Startup.cs

```diff
@@ -44,8 +46,17 @@ public void ConfigureServices(IServiceCollection services)
             });
         }
 
-        public void Configure(IApplicationBuilder app)
+        public void Configure(IApplicationBuilder app, TestLoggerFactoryManager testLoggerFactory)
         {
+            app.Use(async (context, next) =>
```

Comment:
Could we put this in a startup filter and put it in `ServerFixture`? So everybody gets it for free?

Reply:
I'm not checking this in, I'm just trying to find the bugs